internal/feature.go

```diff
@@ -42,10 +42,10 @@ type featureTest struct {
 //
 // The return values have the following semantics:
 //
+//   err == ErrNotSupported: the feature is not available
```

Comment:
Ah, this is the crucial part. :+1: Couldn't deduct from the commit messages which solution you went with eventually.

Reply:
Ah, my bad, thanks!